parts/linux/cloud-init/artifacts/cse_install.sh

```diff
@@ -633,6 +635,29 @@ installKubeletKubectlAndKubeProxy() {
     rm -rf /usr/local/bin/kubelet-* /usr/local/bin/kubectl-* /home/hyperkube-downloads &
 }
 
+generateKubeBinaryRegistryURL() {
```

Comment:
this seems to be an unused duplicate of `updateKubeBinaryRegistryURL`, can we remove it?